static/app/components/timeline/index.tsx

```diff
@@ -0,0 +1,177 @@
+import {useMemo} from 'react';
+import styled from '@emotion/styled';
+
+import {getFormattedTimestamp} from 'sentry/components/events/interfaces/breadcrumbs/breadcrumb/time/utils';
```

Comment:
had no idea this existed but it doesn't seem to support user selected timezones but i see that this what is used int he existing breadcrumb component

Reply:
I'll add this to our feedback db for the project to support user selected timezones later on. Thanks!